### DISCUSSION TITLE
How to disable on-screen mouse information and INFO statements

### DISCUSSION THREAD

**Discussion by stlhoosier97** [aevyrie/bevy_mod_picking#203] [Q&A] [answered]
This is likely a simple question but I can't figure it out for some reason.

How do I disable the diagnostic text that appears over the mouse and the INFO statements that look like this?

`INFO bevy_picking_core::debug: Mouse, Vec2(849.0, 574.0), 65v0, Out { hit: HitData { camera: 2v0, depth: 24.6, position: Some(Vec3(1.6, 0.7, -4.5)), normal: Some(Vec3(0.0, 1.0, 0.0)) } }`

Thanks!

**Comment by aevyrie** [comment, marked as answer]
See the gltf example:

https://github.com/aevyrie/bevy_mod_picking/blob/3dc4c5cb2e18d0794937c885db905aa057a298fb/examples/gltf.rs#L12-L16

**Reply by stlhoosier97** [reply]
This worked great - thank you!

For anyone else coming along, the syntax for disabling the highlighting plugin at the same time is:

`.add_plugins(DefaultPickingPlugins.build().disable::<DefaultHighlightingPlugin>().disable::<DebugPickingPlugin>());`
